.circleci/config.yml

```diff
@@ -3,26 +3,23 @@ jobs:
   test:
     working_directory: ~/calculated_attributes
     docker:
-      - image: circleci/ruby:2.5
+      - image: cimg/ruby:3.1.3
         environment:
           GEM_HOME: ~/calculated_attributes/vendor/bundle
     steps:
       - checkout
       - restore_cache:
-          key: dependency-cache-{{ checksum "calculated_attributes.gemspec" }}
+          key: dependency-cache-1-{{ checksum "calculated_attributes.gemspec" }}
       - run:
-          name: bundle-install
-          command: bundle check || bundle install --jobs=4 --retry=3
-      - run:
-          name: appraisal-install
-          command: bundle exec appraisal install
```

Comment:
If we are dropping the [appraisal gem](https://github.com/thoughtbot/appraisal) in CI, then we should also remove the [development dependency](https://github.com/aha-app/calculated_attributes/blob/DEVOPS-1953-update-gemfile-for-rails7/calculated_attributes.gemspec#L20) from the gemspec as well as removing the entire [Appraisals configuration file](https://github.com/aha-app/calculated_attributes/blob/DEVOPS-1953-update-gemfile-for-rails7/Appraisals).

However, calculated_attributes is publicly available gem that is used by developers outside of Aha! We might want to consider continuing to use appraisal to manage our build matrix and just change the Appraisals config to run rails 6 and 7.